From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 27 Feb 2016 08:37:47 -0700
Subject: update-pylint-expected-values

---
 test/flycheck-test.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/flycheck-test.el b/test/flycheck-test.el
index 9f7644b..bfc05c3 100644
--- a/test/flycheck-test.el
+++ b/test/flycheck-test.el
@@ -4450,7 +4450,7 @@ Why not:
      '(9 5 info "Missing method docstring" :id "missing-docstring" :checker python-pylint)
      '(9 5 warning "Method could be a function" :id "no-self-use"
          :checker python-pylint)
-     '(10 16 warning "Used builtin function 'map'" :id "bad-builtin"
+     '(10 16 warning "Used builtin function 'map'. Using a list comprehension can be clearer." :id "bad-builtin"
           :checker python-pylint)
      '(12 1 info "No space allowed around keyword argument assignment"
           :id "bad-whitespace" :checker python-pylint)
@@ -4483,7 +4483,7 @@ Why not:
      '(9 5 info "Missing method docstring" :id "C0111" :checker python-pylint)
      '(9 5 warning "Method could be a function" :id "R0201"
          :checker python-pylint)
-     '(10 16 warning "Used builtin function 'map'" :id "W0141"
+     '(10 16 warning "Used builtin function 'map'. Using a list comprehension can be clearer." :id "W0141"
           :checker python-pylint)
      '(12 1 info "No space allowed around keyword argument assignment"
           :id "C0326" :checker python-pylint)
