From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 27 Feb 2016 10:00:48 -0700
Subject: hack-run-env-vars

---
 test/run.el | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/run.el b/test/run.el
index 2b68967..9145d97 100755
--- a/test/run.el
+++ b/test/run.el
@@ -119,6 +119,8 @@ Node `(ert)Test Selectors' for information about test selectors."
          (current-file (if load-in-progress load-file-name (buffer-file-name)))
          (source-directory (locate-dominating-file current-file "Cask"))
          (pkg-rel-dir (format ".cask/%s/elpa" emacs-version)))
+    (setenv "LC_ALL" "C")
+
     (setq package-user-dir (expand-file-name pkg-rel-dir source-directory))
     (package-initialize)
 
